src/components/CallView/CallView.vue

```diff
@@ -30,26 +30,23 @@
 			:local-shared-data="localSharedData" />
 
 		<template v-else>
-			<EmptyCallView v-if="!remoteParticipantsCount && !screenSharingActive && !isGrid" :is-sidebar="isSidebar" />
+			<EmptyCallView v-if="!callParticipantModels.length && !screenSharingActive && !isGrid" :is-sidebar="isSidebar" />
 
 			<div id="videos">
 				<template v-if="!isGrid">
 					<!-- Selected video override mode -->
 					<div v-if="showSelectedVideo"
 						class="video__promoted selected-video"
 						:class="{'full-page': isOneToOne}">
```

Comment:
You can remove this div and add the classes in VideoVue

Reply:
it has `position`, `width`, `height` in styles, which depends on html elements nesting. Won't touch it as part of this PR

Reply:
Yes because it relies on the parent div with `id=videos` with position absolute, but there won't be any dependencies from the html elements children I think.